src/txdb.cpp

```diff
@@ -268,6 +268,7 @@ bool CBlockTreeDB::LoadBlockIndexGuts(const Consensus::Params& consensusParams,
     // Load mapBlockIndex
     while (pcursor->Valid()) {
         boost::this_thread::interruption_point();
+        if (ShutdownRequested()) return false;
```

Comment:
I would find it difficult to ACK this prior to #13235, because the use of `ShutdownRequested` would create a circular dependency between `txdb` and `init.cpp` where the `ShutdownRequested` function is implemented. Rebasing onto master solves this 😃.

Reply:
IMO no need to rebase when the necessary change is in master as it will be brought in via the merge. Maybe just call out its relevance to the ack, as you have here.